lib/web_ui/dev/run.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io' as io;
+
+import 'package:args/command_runner.dart';
+
+import 'common.dart';
+import 'utils.dart';
+
+import 'steps/compile_tests_step.dart';
+import 'steps/run_tests_step.dart';
+import 'watcher.dart';
+
+/// Runs build and test steps.
+///
+/// This command is designed to be invoked by the LUCI build graph. However, it
+/// is also usable locally.
+///
+/// Usage:
+///
+///     felt run name_of_build_step
+class RunCommand extends Command<bool> with ArgUtils<bool> {
```

Comment:
Should this class be called BuildCommand instead? seems like this class encapsulates the functionality for build steps.

Reply:
`felt build` is already taken. It means "build host_debug_unopt". In this case "run" makes sense because it does is run some steps. Those steps may be building, but the may also execute tests, and in the future check licenses, run analysis, and so on.